test/test_quantized.py

```diff
@@ -1276,6 +1276,48 @@ def test_qlinear_unpack(self, W, use_channelwise, qengine):
                     W_q.q_zero_point(), W_q_origin.q_zero_point())
 
 class TestQuantizedConv(unittest.TestCase):
+    def _test_qconv_unpack_impl(
+        self, qconv_prepack, qconv_unpack, inputs, strides, pads, channelwise
```

Comment:
Naming the function arguments `qconv_prepack` and `qconv_unpack` can be confusing for readers. Can you name them something like `unpack_fn` and `pack_fn` to distinguish them from the ops?

Reply:
Done.